src/Discord/Internal/Rest/Webhook.hs

```diff
@@ -27,14 +27,15 @@ import Discord.Internal.Types
 
 -- aeson introduced type name for json key (text)
 -- https://github.com/haskell/aeson/issues/881
-# if MIN_VERSION_aeson(2, 0, 0)
-import qualified Data.Aeson.Key as Key
-toKey :: T.Text -> Key.Key
-toKey = Key.fromText
-# else
-toKey :: T.Text -> T.Text
-toKey = id
-# endif
+-- this is unneeded due to version bounds.
+-- # if MIN_VERSION_aeson(2, 0, 0)
+-- import qualified Data.Aeson.Key as Key
+-- toKey :: T.Text -> Key.Key
```

Comment:
Commented out?

Reply:
Commented out because my formatter kept evaluating it weirdly, and we limit the `aeson` version we're using currently.